dbgl.spec

```diff
@@ -1,30 +1,41 @@
-Name:			dbgl
-Summary:	        DOSBox Game Launcher
-URL:			http://home.quicknet.nl/qn/prive/blankendaalr/dbgl/
-Version:		0.80
-Release:		1%{?dist}
-BuildArch:		noarch
-License:		GPLv2
+%global debug_package %{nil}
+
+Name:           dbgl
+Summary:        DOSBox Game Launcher
+URL:            http://home.quicknet.nl/qn/prive/blankendaalr/dbgl/
+Version:        0.80
+Release:        2%{?dist}
+License:        GPLv2
 BuildRequires:  ant
-BuildRequires:	eclipse-swt
-BuildRequires:	hsqldb
-BuildRequires:	jpackage-utils
-BuildRequires:	p7zip
-BuildRequires:	unzip
-BuildRequires:	xerces-j2
-BuildRequires:	liboil
-BuildRequires: desktop-file-utils
-Requires:		dosbox >= 0.70
-Requires:		eclipse-swt
-Requires:		hsqldb
-Requires:               java >= 1:1.7.0
-Requires:               java-headless >= 1:1.7.0
-Requires:		jpackage-utils
-Requires:		SDL_net
-Requires:		SDL_sound
-Source0:		http://members.quicknet.nl/blankendaalr/dbgl/download/src080.zip
-Source2:		%{name}.png
-Source3:		%{name}
+BuildRequires:  eclipse-swt
+BuildRequires:  hsqldb
+#BuildRequires:  jpackage-utils
+#BuildRequires:  p7zip
+#BuildRequires:  unzip
+#BuildRequires:  xerces-j2
+#BuildRequires:  liboil
+BuildRequires:  desktop-file-utils
+BuildRequires:  java-devel
+BuildRequires:  apache-commons-lang3
+BuildRequires:  apache-commons-io
+BuildRequires:  libappstream-glib
+#BuildRequires:  jersey
+#BuildRequires:  glassfish-jax-rs-api
+#BuildRequires:  glassfish-hk2-utils
+#BuildRequires:  glassfish-hk2-configuration
+#BuildRequires:  glassfish-hk2-metadata-generator
+#BuildRequires:  glassfish-hk2-hk2
+Requires:       dosbox >= 0.70
+Requires:       eclipse-swt
+Requires:       hsqldb
+Requires:       java >= 1:1.7.0
+Requires:       java-headless >= 1:1.7.0
+Requires:       jpackage-utils
+Requires:       SDL_net
+Requires:       SDL_sound
+Source0:        http://members.quicknet.nl/blankendaalr/dbgl/download/src080.zip
+Source1:        https://raw.githubusercontent.com/UnitedRPMs/dbgl/master/dbgl.desktop
+Source2:        https://raw.githubusercontent.com/UnitedRPMs/dbgl/master/dbgl.appdata.xml
```

Comment:
Upload this files to your pull request and change URLs to https://github.com/RussianFedora/dbgl/.